packages/core/template/project/TracingCode.md

```diff
@@ -0,0 +1,112 @@
+---
+  layout: default.md
+  title: "Tutorial: Tracing code"
+  pageNav: 3
+---
+
+# Tutorial: Tracing code
```

Comment:
I think this can be shorter

Keep the quote tho since you haven't used quotes elsewhere!